android/src/main/java/com/swmansion/gesturehandler/react/RNGestureHandlerButtonViewManager.kt

```diff
@@ -17,32 +17,39 @@ import android.view.View.OnClickListener
 import android.view.ViewGroup
 import androidx.core.view.children
 import com.facebook.react.bridge.SoftAssertions
-import com.facebook.react.uimanager.PixelUtil
-import com.facebook.react.uimanager.ThemedReactContext
-import com.facebook.react.uimanager.ViewGroupManager
-import com.facebook.react.uimanager.ViewProps
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.uimanager.*
```

Comment:
Do you want to have `*` import here?

Reply:
I changed it to specific imports in https://github.com/software-mansion/react-native-gesture-handler/pull/1893/commits/89232814dc7aa64bda39cb1874016771a99ebfa0. There was a problem where I couldn't import a specific file but `*` made it work, it seems to be ok now.